k3d.yaml

```diff
@@ -59,12 +59,16 @@ subpackages:
           mkdir -p "${{targets.subpkgdir}}"/etc/confd
           cp -ar templates "${{targets.subpkgdir}}"/etc/confd/
           cp -ar conf.d "${{targets.subpkgdir}}"/etc/confd/
-      - uses: git-checkout
+      - name: git-chekout-confd # There is not a new release in last 6 years
+        runs: |
+          git clone https://github.com/kelseyhightower/confd.git /home/confd
```

Comment:
I would still use the git-checkout pipeline if possible, we can specify `branch` too.

I'd be comfortable building from HEAD for this. Looks like a somewhat unmaintained a fork, but I'm very comfortable pulling from Kelsey

Reply:
Ah - looks like `iwilltry42` is a fork from Kelsey, I had missed that